### PR TITLE
6.article id comments

### PR DESCRIPTION
**Pull request by cgtorrubia**
Task 6. Check Comments by Article ID done.

I have dropped an Easter Egg that I found funny to make it a bit "different" than the usual.

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -86,6 +86,70 @@ describe('GET /api/articles/:article_id', () => {
...
+	})
+	test('GET - Easter Egg', () => {
+		return request(app)
+			.get('/api/articles/teapot/comments')
+			.expect(418)
+			.then(({ body: { msg } }) => {
+				expect(msg).toBe("Hi, I'm just a tiny teapot!")
+			})
+	})
+})
```
Comment (johnhutcheon) [inline __tests__/app.test.js:161]: Really nice testing! Could you just add one more test for- Status 200, valid ID, but has no comments responds with an empty array of comments?

**Review by johnhutcheon — CHANGES_REQUESTED** [review]
Really nice work Cris! I love the Easter Egg 😄 Could you just add the further test as per my comment please and resubmit? Thanks!

**Review by B7Bprog — APPROVED** [review]
Hi Cristóbal, Nice refactoring. Love the easter egg too ;) Please merge!
